Makefile

```diff
@@ -27,8 +27,11 @@ else
 	GOPATH := $(shell pwd)/_gopath
 endif
 
+
+# Use system python if it exists, otherwise use Docker.
+PYTHON := $(shell command -v foo || docker run --rm -it -v $(shell pwd):/minikube -w /minikube python python)
```

Comment:
Should it really be `command -v foo`?
I would imagine python somewhere in that command :)